app/models/mandate.js

```diff
@@ -2,6 +2,6 @@ import Model, {hasMany, belongsTo } from '@ember-data/model';
 
 export default class MandateModel extends Model {
     @belongsTo('board-position') roleBoard;
-    @belongsTo('governing-body', { inverse: true }) governingBody;
```

Comment:
`inverse: true` isn't valid. It should either be the name of the relationship in the linked model instead.